components/brave_rewards/resources/extension/brave_rewards/adaptive_captcha_panel.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+
```

Comment:
I think we should probably add:

```ts
import 'emptykit.css'
import '../../../../../ui/webui/resources/fonts/muli.css'
import '../../../../../ui/webui/resources/fonts/poppins.css'
```

To make sure that we're pulling in the CSS reset and the right fonts. Ideally, we wouldn't be bundling these resources, but I'm not sure how to get around it for extension bundles right now.

Reply:
Done.